charts/k0s/templates/resourcequota.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.isolatedMode.enabled }}
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name:  {{ .Release.Name }}-quota
+spec:
+  hard:
+    cpu: {{ .Values.isolatedMode.resourceQuota.cpu | quote }}
+    memory: {{ .Values.isolatedMode.resourceQuota.memory }}
+    pods: {{ .Values.isolatedMode.resourceQuota.pods | quote }}
```

Comment:
Please add an option here to add other fields

Reply:
PTAL